src/app/services/loading/loading.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+
+import { BehaviorSubject, Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class LoadingService {
+  private loadingSubject$$ = new BehaviorSubject(false);
```

Comment:
I believe, that `Subject` part can be skipped

Reply:
@boale  To be honest, I don't completely understand how it can be skipped, because without `loadingSubject$$` the rest of methods in this service simply won't work. Am I mistaken?

Reply:
I ment that `loading$$` is any type of Subject by convention.

Reply:
Got you